src/app/components/shell/topbar/topbar.component.ts

```diff
@@ -10,12 +11,13 @@ import { Component } from '@angular/core';
 })
 export class TopbarComponent {
   userProfileItems: MenuItem[] = [
-    { label: 'Logout', icon: 'pi pi-sign-out', routerLink: ['/onboarding/login'] },
-    { label: 'Profile', icon: 'pi pi-user-edit', routerLink: ['/onboarding/profile'] }
+    { label: this.translate.instant('logout.LOG_OUT'), icon: 'pi pi-sign-out', routerLink: ['/onboarding/logout'] },
```

Comment:
das kapiere ich noch nicht mit dem "instant".. was passiert hier genau?

Reply:
translate pipe kann man nur in der html Datei benutzen; translate.instant kann man benutzten wenn man etwas in der typescript Datei übersetzen will